.github/promote-images.yml

```diff
@@ -38,7 +38,7 @@
     workspace-java-11: "TIMESTAMP_TAG"
     workspace-java-17: "TIMESTAMP_TAG"
     workspace-java-21: "TIMESTAMP_TAG"
-    workspace-java-23: "TIMESTAMP_TAG"
+    workspace-java-25: "TIMESTAMP_TAG"
```

Comment:
Hi @rajadilipkolli , we cannot remove workspace-java-23, some folks may still depend on it. I'll open a separate PR to introduce java 25, though.

Reply:
Hi @rajadilipkolli , the PR for Java 25 has landed, the main build takes roughly 1h, after which you should see the new image on Docker Hub, with a time stamped tag.